plugins/html-elements-plugin/src/main/java/ru/sbtqa/tag/pagefactory/reflection/HtmlReflectionImpl.java

```diff
@@ -115,7 +169,7 @@ public static boolean isChildOf(Class<?> parent, Field field) {
      * @return list of found blocks. could be empty
      * @throws IllegalAccessException if called with invalid context
      */
-    private static List<HtmlElement> findBlocks(String blockPath, Object context, boolean returnFirstFound)
+    private List<HtmlElement> findBlocks(String blockPath, Object context, boolean returnFirstFound)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this method to reduce its Cognitive Complexity from 19 to the 15 allowed. ([squid:S3776](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS3776))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=101&issues=AWbDPCupSgN4uCclZPgb&open=AWbDPCupSgN4uCclZPgb)

[ISSUE_KEY]: AWbDPCupSgN4uCclZPgb